### PR TITLE
Fix: trickle incorrectly unlocking when switching languages

### LINKED ISSUE #412: Trickle unlocking when switching languages
### Subject of the issue/enhancement/features
If a page contains more than one article, with trickle applied to the first article, trickle will be incorrectly removed when switching languages. This only seems to occur when the course has an enabled `_start` location.

### Your environment
* Framework 5.31.12 but is specific to a fix required for https://github.com/adaptlearning/adapt-contrib-trickle/issues/195

### PR DESCRIPTION
**Pull request by danielghost**
Fixes #412.

[//]: # (Delete Fix, Update, New and/or Breaking sections as appropriate)
### Fix
* Trickle incorrectly unlocking when switching languages

### Update
* Minor code refactor
* Reduced routing on launch

### CODE REVIEW

File: js/startController.js [hunk 1]
```diff
@@ -8,18 +7,35 @@ class StartController extends Backbone.Controller {
 
   initialize(...args) {
     super.initialize(...args);
+    this._isSessionInProgress = false;
     this.model = null;
+    this.setupListeners();
+  }
+
+  setupListeners() {
+    this.listenTo(Adapt, {
```
[the remark this answers is not in the document]
Comment (danielghost) [reply js/startController.js:16]: The Class is only initialised once, so the listeners need to account for multiple triggers, e.g. when switching languages.

**Review by eleanor-heath — APPROVED** [review]

**Review by joe-allen-89 — APPROVED** [review]
👍

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Pull request merged**